zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/multitenancy/TenantAwareCancelProcessInstanceTest.java

```diff
@@ -61,40 +60,6 @@ public void shouldCancelInstanceForDefaultTenant() {
         .hasIntent(ProcessInstanceIntent.ELEMENT_TERMINATED);
   }
 
-  @Test
-  public void shouldRejectCancelInstanceForUnauthorizedTenant() {
```

Comment:
💭 I didn't found a way to test this with the new tenant check. If you have ideas please let me know. 
I want to add some IT test for this, same as the authorization ITs, but I need to wait until we have the tenant command on the client

Reply:
The test still seems valid to me. What's the problem in running it? Is it that authorizations is not enabled on the engine? If so I'm confident that we can find a way to enable this.

Reply:
Yes, that's the reason

Reply:
Happy to pair with you to get authorizations enabled in the engine rule. Just ping me 🙂

Reply:
@nicpuppa I made a quick example on what you need to change in the `EngineRule` to allow enabling authorizations. You can find it [here](https://github.com/camunda/camunda/commit/a4b5306ec6c1946d4e1593a713ec7949a2588a29).

With that you can enable it like:

```java
  @ClassRule
  public static final EngineRule ENGINE =
      EngineRule.singlePartition()
          .withSecurityConfig(config -> config.getAuthorizations().setEnabled(true));
```

Reply:
Note that this just a quick hack and doesn't allow you to set multiple configurations, as calling `withSecurirtyConfig` a second time would override it. But it should give you an idea.